README.md

```diff
@@ -80,11 +96,61 @@ When this element is rendered, the DOM will look something like the following:
 </x-hello>
 ```
 
-This is the utility that web components provide when using Custom Elements and the Shadow DOM.
+This is the utility that web components provide when using Custom Elements and
+the Shadow DOM.
+
+Skate also allows **turning off Shadow DOM** if you don't wanna use it for
+various particular reasons. You can turn it off via `get renderRoot()` override:
+
+> NOTE: by turning off Shadow DOM you cannot use <slot/> content projection
+> anymore by default, further tweaks needs to be applied
+
+```js
+import { withComponent, props } from 'skatejs';
+
+// define base class withouth Shadow DOM
+class NoShadowComponent = class extends withComponent() {
+  // you need to return where you want to render your content, in our case we wanna render directly to our custom element children
```

Comment:
The remainder of the lines in this diff fit in the 80 char width, except for this one </nitpick>